plans/create.pp

```diff
@@ -60,8 +66,8 @@
 #   The type of operating system (linux or windows) to be used for provisioning the VMs
 #
 plan provision::create(
-  String[1] $subnet,
-  Array[String[1]] $security_group_ids,
+  Optional[String[1]] $subnet = undef,
+  Optional[Array[String[1]]] $security_group_ids = ['x'],
```

Comment:
we should either keep it undef or an empty array, there should not be any values into that.

Reply:
sure.